paper/paper.Rmd

```diff
@@ -0,0 +1,171 @@
+---
+title: "Static Code Analysis for R"
+date: "`r Sys.Date()`"
+tags: ["R", "linter", "tidyverse"]
+authors:
+  - name: Jim Hester
+    affiliation: 1
+    orcid: ~
+  - name: Michael Chirico
+    affiliation: 2
+    orcid: ~
+  - name: Alexander Rosenstock
+    affiliation: 3
+    orcid: ~
+  - name: Indrajeet Patil
+    orcid: 0000-0003-1995-6531
+    affiliation: 4
+affiliations:
+  - index: 1
+    name: Netflix
+  - index: 2
+    name: Google
+  - index: 3
+    name: Mathematisches Institut der Heinrich-Heine-Universität Düsseldorf
+  - index: 4
+    name: Center for Humans and Machines, Max Planck Institute for Human Development, Berlin, Germany
+output: 
+    md_document:
+      variant: "markdown"
+      preserve_yaml: true
+      standalone: true
+bibliography: paper.bib
+csl: apa.csl
+link-citations: yes
+---
+
+```{r, warning=FALSE, message=FALSE, echo=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  out.width = "100%",
+  comment = "#>"
+)
+
+library(lintr)
+
+withr::local_options(list(
+  lintr.format_width = 60L
+))
+```
+
+# Statement of Need
+
+R is an interpreted, dynamically typed programming language [@base2023]. It is a popular choice for statistical analysis and visualization, and is used by a wide range of researchers and data scientists. The `{lintr}` package is an open-source R package that provides static code analysis to check for a variety of common problems related to readability, efficiency, consistency, style, etc. In particular, it enforces the [tidyverse style guide]((https://style.tidyverse.org/index.html)). It is designed to be easy to use and integrate into existing workflows, and can be run from the command line or used as part of an automated build or continuous integration process. `{lintr}` also integrates with a number of popular IDEs and text editors, such as RStudio and Visual Studio Code, making it convenient for users to run `{lintr}` checks on their code as they work.
```

Comment:
> can be run from the command line

This might make it seem like one could do `./lintr file --linters=assignment_linter()` or something like that, but we don't ship any command-line binary, you have to invoke `Rscript` (right?)

Reply:
True. I no longer mention the command line here.